closed/get_j9_source.sh

```diff
@@ -57,10 +57,16 @@ declare -A shas
 declare -A references
 
 git_urls[openj9]=https://github.com/eclipse-openj9/openj9
-branches[openj9]=master
-
 git_urls[omr]=https://github.com/eclipse-openj9/openj9-omr
-branches[omr]=openj9
+
+currentbranch=`git rev-parse --abbrev-ref HEAD`
+if [[ "$currentbranch" =~ v[0-9]+\.[0-9]+[\.[0-9]+]?-release ]] ; then
```

Comment:
I think we need to use `()` instead of `[]` for the optional third version component:
```
if [[ "$currentbranch" =~ v[0-9]+\.[0-9]+(\.[0-9]+)?-release ]] ; then
```